docs/dockerfile-frontend-options.md

```diff
@@ -0,0 +1,65 @@
+# Dockerfile Frontend Build Options
+
+## use with `buildctl build --opt`
+For `dockerfile.v0` frontend there are some parameters that can be used with `buildctl build --opt`.
+The following parameters are the most frequently used.
+* target: set the target build stage to build.
```

Comment:
This doesn't render that well. I think they should be all separate paragraphs. Eg. look at https://github.com/docker/buildx/blob/master/docs/reference/buildx_build.md

Reply:
We should order them based on priority. For each one we should also show the equivalent in docker, eg. `docker buildx build --target foo`